src/DashboardWidgetItem.vue

```diff
@@ -240,7 +243,11 @@ export default {
 	},
 
 	methods: {
-
+		onLinkClick(event) {
+			if (event.target.tagName === 'BUTTON') {
```

Comment:
Not entirely convinced about this, since a click might also be triggered by some other element of course and should not be catched then. Maybe it is a better approach to call event.stopPropagation on the child nodes that should not bubble events further?

Reply:
I completely agree. So the fix should be done in nextcloud-vue "Actions" component. It seems this problem is only happening since "Actions" is using Popover. I'll create an issue in nextcloud-vue because I don't find a simple fix.